arrus/core/devices/us4r/us4oem/Us4OEMDescriptorFactory.h

```diff
@@ -77,6 +77,36 @@ class Us4OEMDescriptorFactory {
                 },
                 4 // maximum number of timeouts
             };
+        case 3:
+            // us4OEM+ variant 0
+            return Us4OEMDescriptor{
+                version, // us4OEM version
+                32, // RX channels
+                20e-6f,  // min. RX time,
+                5e-6f, // RX time epsilon,
+                35e-6f, // TX parameters reprogramming time,
+                120e6f, // Sampling frequency [Hz]
```

Comment:
Sampling frequency should depend on AFE(OEM) variant

Reply:
This is the only place where the 120 MHz is right now stored -- as a property of the OEM HF descriptor. So this line is OK.